devTemplate.html

```diff
@@ -1,16 +1,16 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <script>
       window.__ACTION__ = <%= htmlWebpackPlugin.options.__ACTION__ %>
     </script>
-    <title>Free Online Retrospectives | Parabol</title>
+    <title>Parabol | Streamline or Replace Meetings</title>
```

Comment:
-1 This breaks `AnalyticsPage` page title extraction logic
https://github.com/ParabolInc/parabol/blob/bf252ffbebdef2805995e5ae3337144b22b55aab/packages/client/components/AnalyticsPage.tsx#L174
In general the PR is not consistent with the part before and after `|`, it should follow some hierarchy I think, probably most specific first.

Reply:
I didn't understand why this design reversed these components. I figured people had some sort of logic as to why they made this choice, but now that I see there is other logic that is dependent on the ordering here I'll make this consistent